buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -391,15 +391,20 @@ class ClusterFormationTasks {
         return configureExecTask(name, project, setup, node, args)
     }
 
-    /** Surround strings that contains a comma with double quotes **/
-    private static String escapeComma(Object o) {
-        if (o instanceof String) {
-            String s = (String)o
+    /** Wrapper for command line argument: surrounds comma with double quotes **/
+    static class ExecArgWrapper {
```

Comment:
I would call this something more specific, like EscapeCommaWrapper, and also it can be private?